CorsixTH/Lua/entities/humanoids/staff.lua

```diff
@@ -670,19 +670,19 @@ function Staff:getDrawingLayer()
   return 4
 end
 
---! Estimate staff service quality based on skills, fatigue and happiness.
+--! Estimate staff service quality based on skills, restfulness (inverse of fatigue) and happiness.
 --!return (float) between [0-1] indicating quality of the service.
 function Staff:getServiceQuality()
   -- weights
   local skill_weight = 0.7
-  local fatigue_weight = 0.2
+  local rest_weight = -0.2 -- Less fatigue is better
```

Comment:
With this new variable, we should probably protect against the service quality going <0. It's highly unlikely but could happen if you have a verrrryyy tired unskilled staff member who's miserable.

Reply:
Also that now this is a negative number we the variable name may need a tweak

Reply:
The function doc says [0-1] so a minimum of 0 will be enforced.